vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/WebDriverSetupSteps.java

```diff
@@ -52,8 +54,25 @@ public void beforeStory()
         processMeta(bddRunContext.getRunningStory().getStory().getMeta());
     }
 
+    @AfterScenario(uponType = ScenarioType.ANY)
+    public void afterScenario()
+    {
+        if (webDriverLifeScope == WebDriverLifeScope.SCENARIO)
+        {
+            stopWebDriver();
+        }
+    }
+
     @AfterStory
     public void afterStory()
+    {
+        if (webDriverLifeScope == WebDriverLifeScope.STORY || webDriverLifeScope == null)
```

Comment:
I propose to throw an exception in case of we got null, because it's not clear what's the actual behavior user expecting